Makefile

```diff
@@ -87,9 +87,16 @@ BASE_IMAGE ?= alpine
 BASE_IMAGE_TAG ?= 3.20
 
 # Local Development
-KIND_CLUSTER ?= yhs
+CLUSTER_MGR ?= kind     # either 'kind' or 'minikube'
+CLUSTER_NAME ?= yhs
 NAMESPACE ?= yunikorn
 
+KIND ?= $(LOCALBIN_TOOLING)/kind
+KIND_VERSION ?= v0.23.0
```

Comment:
Why not "latest"?

Reply:
Good suggestion - it was at that fixed version before, but it's safe to use 'latest' - kind is very heavily tested before every release. Fixed in commit 672c721.